MapboxMobileEvents/CLLocationManager+MMEMobileEvents.m

```diff
@@ -0,0 +1,35 @@
+#import "CLLocationManager+MMEMobileEvents.h"
+#import "MMEConstants.h"
+
+@implementation CLLocationManager (MMEMobileEvents)
+
+void mme_linkCLLocationManagerCategory(){}
+
++ (NSString *)locationAuthorizationStatus {
```

Comment:
suggest renaming to `mme_authorizationStatusString` to prevent potential namespace collision and match to the underlying call name more closely

Reply:
Good point